packages/host/app/commands/add-skills-to-room.ts

```diff
@@ -53,7 +53,8 @@ export default class AddSkillsToRoomCommand extends HostBaseCommand<
             (newCommandDefinition) =>
               !commandDefinitions.some(
                 (commandDefinition) =>
-                  commandDefinition.name === newCommandDefinition.name,
+                  commandDefinition.sourceUrl ===
+                  newCommandDefinition.sourceUrl,
```

Comment:
I set the sourceUrl of the command definition using codeRef, as it seems more reliable than using name, which can be duplicated.